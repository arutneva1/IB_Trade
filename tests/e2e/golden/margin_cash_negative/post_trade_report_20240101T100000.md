| symbol | side | filled_shares | avg_price | notional | avg_slippage | residual_drift_bps |
| --- | --- | --- | --- | --- | --- | --- |
| BBB | SELL | -20.0000 | 49.00 | -980.00 | 0.00 | 22.08 |
| AAA | SELL | -40.0000 | 99.00 | -3960.00 | 0.00 | 88.30 |
