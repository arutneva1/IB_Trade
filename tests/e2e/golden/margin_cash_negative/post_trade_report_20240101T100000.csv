symbol,side,filled_shares,avg_price,notional,avg_slippage,residual_drift_bps
BBB,SELL,-20.0,49.0,-980.0,0.0,22.08
AAA,SELL,-40.0,99.0,-3960.0,0.0,88.3
